tests/integration/models/reddit/test_live.py

```diff
@@ -42,3 +44,89 @@ def test_updates(self, _):
             for update in thread.updates(limit=None):
                 assert update.thread == thread
         assert update.body.startswith('Small change:')
+
+
+class TestLiveContributorRelationship(IntegrationTest):
+    @mock.patch('time.sleep', return_value=None)
+    def test_invite__already_invited(self, _):
+        self.reddit.read_only = False
+        thread = LiveThread(self.reddit, 'xyu8kmjvfrww')
+        with self.recorder.use_cassette(
+                'TestLiveContributorRelationship_'
+                'test_invite__already_invited'):
+            thread.contributor.invite('nmtake')
+            with pytest.raises(APIException) as excinfo:
+                thread.contributor.invite('nmtake')
+        assert excinfo.value.error_type == 'LIVEUPDATE_ALREADY_CONTRIBUTOR'
+
+    @mock.patch('time.sleep', return_value=None)
+    def test_invite__empty_list(self, _):
+        self.reddit.read_only = False
+        thread = LiveThread(self.reddit, 'xyu8kmjvfrww')
+        with self.recorder.use_cassette(
+                'TestLiveContributorRelationship_test_invite__empty_list'):
+            thread.contributor.invite('nmtake', [])
+
+    @mock.patch('time.sleep', return_value=None)
+    def test_invite__limited(self, _):
+        self.reddit.read_only = False
+        thread = LiveThread(self.reddit, 'xyu8kmjvfrww')
+        with self.recorder.use_cassette(
+                'TestLiveContributorRelationship_test_invite__limited'):
+            thread.contributor.invite('nmtake', ['manage', 'edit'])
+
+    @mock.patch('time.sleep', return_value=None)
+    def test_invite__name(self, _):
```

Comment:
Would you consider this test redundant as other tests rely on the invite by name feature?